spmd/tensor/ops/common_rules.py

```diff
@@ -326,11 +326,12 @@ def reduction_rule(op_schema: OpSchema) -> OutputSharding:
         keep_dim = len(op_schema.args_schema) > 2 and bool(
             op_schema.args_schema[2]
         )
-        dim_list = as_list(op_schema.args_schema[1])
+        dims_list = cast(List[int], as_list(op_schema.args_schema[1]))
+        dim_list = normalize_dims(dims_list, input_spec.ndim)
```

Comment:
nit: is there a reason to keep dims_list around before normalization? is that used below? if not , better remove to avoid confusion